package.json

```diff
@@ -10,9 +10,10 @@
   ],
   "author": "WalletConnect <walletconnect.org>",
   "homepage": "/testwallet",
+  "homepage-broken": ".",
   "license": "LGPL-3.0",
   "scripts": {
-    "start": "react-scripts-ts start",
+    "start": "serve -s build",
```

Comment:
Maybe make this a new script endpoint called `serve` rather than removing the ability to run it in dev mode?

Reply:
I'm going to add a -dev script.